src/funmirtar/utils/plots.py

```diff
@@ -0,0 +1,200 @@
+import pandas as pd
+import numpy as np
+import seaborn as sns
+import matplotlib.pyplot as plt
+from sklearn.metrics import r2_score
+from sklearn.feature_selection import r_regression
+from scipy.stats import spearmanr
+from scipy.stats import pearsonr
+from sklearn.metrics import precision_recall_fscore_support, PrecisionRecallDisplay
+
+
+def plot_multiple_cdfs_with_medians(
+    dataframes_for_top_preds=[], labels_for_top_preds=[], 
+    values_list_for_all_lfc=[], labels_for_all_flc=[], 
+    title_sufix='', num_of_top_preds=16
+):
+    """
+    Plots the Cumulative Density Function (CDF) of log2 fold change (LFC) values with median lines for multiple sets of data.
+    Each line represents a different dataset, showing the proportion of LFC values below a given threshold.
+    The difference is that values_list_for_all_lfc FC values are plotted as a whole, whereas for dataframes_for_top_preds we pick the top 16 predictions and plot only their LFC.
+    It is possible to plot only one group (top 16 or all LFC values) and leave the other empty.
+
+    Parameters:
+    - dataframes_for_top_preds: List of DataFrames, each containing 'miRNA', predictions (denoted by label), and fold changes (denoted by 'fold_change'). Only top N predictions for each miRNA are picked and their LFC is plotted. 
+    - labels_for_top_preds: List of prediction labels, one label for each method
+    - values_list_for_all_lfc: List of arrays of LFC values, all values will be plotted
+    - labels_for_all_flc: List of labels_for_all_flc for values_list_for_all_lfc, one label for each LFC array
+    - title_sufix: To be used in plot title
+    - num_of_top_preds: Number of top predictions for each miRNA
```

Comment:
Can we select number of top predictions? In description, you are mentioning fixed number 16

Reply:
Changed function input params.